api/controllers/v0.2/createPatron.js

```diff
@@ -29,6 +88,14 @@ let ilsClientPassword;
  * @return {object}
  */
 function collectErrorResponseData(status, type, message, title, debugMessage) {
+  logger.error(
+    `status_code: ${status}, ` +
```

Comment:
Does node have a way to build strings with interpolating, rather than concatenating?  Is it generally considered safer?

Reply:
@daryanypl yes Node has a way.  Each of these lines is doing interpolation and then all the lines are concatenated.

We are writing to a log on AWS rather than a database so I think the security considerations are negligible.  I don't know of a security difference between concatenation vs. interpolation.